### PR TITLE
Feature/5233 message widget

### PR DESCRIPTION
**Pull request by nkylstad**
https://github.com/Altinn/altinn-studio/issues/5233

### CODE REVIEW

File: schemas/json/layout/layout.schema.v1.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
             "type": "string",
             "title": "Type",
             "description": "The component type.",
-            "enum": ["AddressComponent", "Button", "Checkboxes", "Datepicker", "Dropdown", "FileUpload", "Group", "Header", "Input", "NavigationButtons", "Paragraph", "RadioButtons", "Summary", "TextArea"]
+            "enum": ["AddressComponent", "Button", "Checkboxes", "Datepicker", "Dropdown", "FileUpload", "Group", "Header", "Input", "NavigationButtons", "Paragraph", "RadioButtons", "Summary", "TextArea", "AttachmentList"]
```
Comment (altinnadmin) [inline schemas/json/layout/layout.schema.v1.json:44]: Should be second in the list (alphabetical order).

File: schemas/json/layout/layout.schema.v1.json [hunk 2]
```diff
@@ -247,6 +248,16 @@
...
           }
         }
+      },
+      "attachmentListComponent": {
+        "properties": {
+          "dataTypeIds": {
+            "type": "array",
+            "title": "Data type IDs",
+            "description": "List of data type IDs for the attachment list to show.",
+            "examples": [["SomeDataType", "SomeOtherDataType"]]
```
Comment (altinnadmin) [inline schemas/json/layout/layout.schema.v1.json:258]: Is the double array intended?

Comment (altinnadmin) [reply schemas/json/layout/layout.schema.v1.json:258]: Missing type definition for array items?
```json
"items": {
      "type": "string"
 }
```

**Review by altinnadmin — CHANGES_REQUESTED** [review]
Nice! Some minor comments :)

File: schemas/json/layout/layout.schema.v1.json (hunk 1 above)
Comment (nkylstad) [reply schemas/json/layout/layout.schema.v1.json]: Fixed :)

File: schemas/json/layout/layout.schema.v1.json (hunk 2 above)
Comment (nkylstad) [reply schemas/json/layout/layout.schema.v1.json:261]: `"examples"` an array of examples, thus the double array :)  
image

Fixed missing type definition for array items

**Review by altinnadmin — APPROVED** [review]
:shipit:

**Pull request merged**